elyra/metadata/schemas/code-snippet.json

```
@@ -19,20 +19,25 @@
       "properties": {
         "description": {
           "description": "Code snippet description",
-          "type": "string"
+          "type": "string",
+          "ui_hints": {
```

Sorry, but I just discovered the use of [`uihints` in properties.json](https://github.com/elyra-ai/elyra/blob/master/packages/pipeline-editor/src/properties.json#L31) and I think we should use the same format here as well (although I think `ui_hints` is better, possession is nine-tenths of the law :smile:).
```suggestion
          "uihints": {
```

---

ill update it